app/src/main/java/com/sun_asterisk/weather_03/MainActivity.java

```diff
@@ -1,13 +1,48 @@
 package com.sun_asterisk.weather_03;
 
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.MenuItem;
+import com.sun_asterisk.weather_03.screen.home.Home_Fragment;
 
 public class MainActivity extends AppCompatActivity {
-
+    Home_Fragment mHome_fragment;
```

Comment:
check access modifier  
```suggestion
    Home_Fragment mHomeFragment;
```